policyengine_us/parameters/gov/states/wa/tax/income/in_effect.yaml

```diff
@@ -1,12 +1,22 @@
 description: Washington applies its income tax rules if this is true.  
 values:
   0000-01-01: false
-  2022-01-01: true
+  1932-11-01: true  # Initiative 69 passed by voters
+  1933-09-08: false  # Culliton v. Chase - Supreme Court ruled unconstitutional
+  2022-01-01: true  # Capital gains tax (upheld as excise tax, not income tax)
 metadata:
   unit: bool
   period: year  
   label: Washington income tax rules in effect
   reference:
+    - title: Culliton v. Chase, 174 Wash. 363 (1933)
+      href: https://law.justia.com/cases/washington/supreme-court/1933/23566-1.html
```

Comment:
reference not found

Reply:
cant open most of these - I think we should be more explicit about referencing as well